### PR TITLE
Ruby 2.6.1 Circle CI image.

### PR DESCRIPTION
**Pull request by usamah82**
Ruby 2.6.1 Circle CI image for the Ruby 2.6.1 upgrade work.

### CODE REVIEW

File: circleci-ruby/2.6.1-stretch-pgclient10.5-node10.13-browsers/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+  esac \
+  && curl -fsSLO --compressed "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-$ARCH.tar.xz" \
+  && curl -fsSLO --compressed "https://nodejs.org/dist/v$NODE_VERSION/SHASUMS256.txt.asc" \
+  && gpg --batch --decrypt --output SHASUMS256.txt SHASUMS256.txt.asc \
+  && grep " node-v$NODE_VERSION-linux-$ARCH.tar.xz\$" SHASUMS256.txt | sha256sum -c - \
+  && tar -xJf "node-v$NODE_VERSION-linux-$ARCH.tar.xz" -C /usr/local --strip-components=1 --no-same-owner \
+  && rm "node-v$NODE_VERSION-linux-$ARCH.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt \
+  && ln -s /usr/local/bin/node /usr/local/bin/nodejs
+
+ENV YARN_VERSION 1.12.3
```
Comment (mutablestate) [inline circleci-ruby/2.6.1-stretch-pgclient10.5-node10.13-browsers/Dockerfile]: @usamah82 yarn stable is currently at `1.13.0` we can upgrade here.

File: circleci-ruby/2.6.1-stretch-pgclient10.5-node10.13-browsers/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+  56730D5401028683275BD23C23EFEFE93C4CFFFE \
+  77984A986EBC2AA786BC0F66B01FBB92821C587A \
+  8FCCA13FEF1D0C2E91008E09770F7A9A5AE15600 \
+  ; do \
+  gpg --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys "$key" || \
+  gpg --keyserver hkp://ipv4.pool.sks-keyservers.net --recv-keys "$key" || \
+  gpg --keyserver hkp://pgp.mit.edu:80 --recv-keys "$key" ; \
+  done
+
+ENV NODE_VERSION 10.13.0
```
Comment (mutablestate) [inline circleci-ruby/2.6.1-stretch-pgclient10.5-node10.13-browsers/Dockerfile]: @usamah82 Node LTS is currently at `10.15.1` we can upgrade here.

**Review by mutablestate — CHANGES_REQUESTED** [review]
@usamah82 we can upgrade Node and Yarn to the latest stable versions at the same time (see inline comments). You'll also need to update the image name to reflect these changes.

File: circleci-ruby/2.6.1-stretch-pgclient10.5-node10.13-browsers/Dockerfile (hunk 1 above)
Comment (usamah82) [reply circleci-ruby/2.6.1-stretch-pgclient10.5-node10.13-browsers/Dockerfile]: Will upgrade yarn. Thanks for the info!

File: circleci-ruby/2.6.1-stretch-pgclient10.5-node10.13-browsers/Dockerfile (hunk 2 above)
Comment (usamah82) [reply circleci-ruby/2.6.1-stretch-pgclient10.5-node10.13-browsers/Dockerfile]: Will upgrade Node. Thanks for the info!

**Comment by usamah82** [comment]
@mutablestate I've upgraded Node and yarn here https://github.com/blacktangent/docker-images/pull/3/commits/b4b595a0484011eb2f25b834b686f2426f34d76f

**Review by mutablestate — APPROVED** [review]
@usamah82 thanks! Looks good.

**Pull request merged**